cmd/antrea-service/main.go

```diff
@@ -0,0 +1,327 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build windows
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/kardianos/service"
+	"golang.org/x/sys/windows"
+)
+
+const (
+	systemPath = "C:\\Windows\\System32\\;C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\;"
+
+	nodeNameEnvKey = "NODE_NAME"
```

Comment:
Should we change this one to argument too? It might be a bigger change (to populate the argument from env variable when Agent runs as a Pod), so I am fine with using env for now if no easy way to change the Agent Pod case.

Reply:
At least you can add some comments to say this is to populate the NODE_NAME env variable to Agent, to keep the same behavior when running Agent as a Pod?

Reply:
Sure, I will add comments.